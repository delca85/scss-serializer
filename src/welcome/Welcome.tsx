import './welcome.scss';

export const Welcome = () => (
  <div className="container">
    <h1 className="welcome">Hello Pippo! </h1>
    <button>Pippo button</button>
  </div>
);
