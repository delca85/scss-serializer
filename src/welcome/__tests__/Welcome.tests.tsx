import React from 'react';
import renderer from 'react-test-renderer';
import { Welcome } from '../Welcome';

describe('Welcome component tests', () => {
  it('should render properly', () => {
    const component = renderer.create(<Welcome />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
