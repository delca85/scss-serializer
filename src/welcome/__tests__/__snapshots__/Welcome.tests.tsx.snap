// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome component tests should render properly 1`] = `
<h1
  className="welcome"
>
  Hello Pippo! 
</h1>
`;
